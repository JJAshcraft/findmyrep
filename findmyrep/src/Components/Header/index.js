import React, { Fragment } from "react";
import styled from "styled-components";
import H1 from "../../Components/H1";
import HR from "../../Components/HR";
const HeaderH1 = styled(H1)`
  padding-bottom: 45px;
  font-weight: 700;
`;

const Header = props => {
  return (
    <Fragment>
      <HeaderH1>Who's My Representative?</HeaderH1>
      <HR />
    </Fragment>
  );
};

export default Header;
