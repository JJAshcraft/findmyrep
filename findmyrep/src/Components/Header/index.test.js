import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import Header from "./index.js";

test("Header component renders with proper styling", () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
