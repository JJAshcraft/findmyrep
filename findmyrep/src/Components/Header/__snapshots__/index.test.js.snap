// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders with proper styling 1`] = `
Array [
  .c0 {
  font-size: 20px;
  font-weight: 500;
  color: #06a7ed;
  padding-bottom: 45px;
  font-weight: 700;
}

<h1
    className="c0"
  >
    Who's My Representative?
  </h1>,
  .c0 {
  border: 1px solid #eee;
}

<hr
    className="c0"
  />,
]
`;
